pkg/url/url.go

```diff
@@ -0,0 +1,73 @@
+package url
+
+import (
+	"fmt"
+
+	"github.com/pkg/errors"
+	"github.com/redhat-developer/ocdev/pkg/application"
+	"github.com/redhat-developer/ocdev/pkg/component"
+	"github.com/redhat-developer/ocdev/pkg/occlient"
+	log "github.com/sirupsen/logrus"
+)
+
+type URL struct {
+	Name string
+	URL  string
+}
+
+// Delete deletes a URL
+func Delete(client *occlient.Client, name string) error {
+	return client.DeleteRoute(name)
+}
+
+// Create creates a URL
+func Create(client *occlient.Client, cmp string) (*URL, error) {
+
+	app, err := application.GetCurrentOrDefault(client)
+	if err != nil {
+		return nil, errors.Wrap(err, "unable to get current application")
+	}
+
+	labels, err := component.GetLabels(cmp, app, false)
+	if err != nil {
+		return nil, errors.Wrap(err, "unable to get labels")
+	}
+
+	route, err := client.CreateRoute(cmp, labels)
+	if err != nil {
+		return nil, errors.Wrap(err, "unable to create route")
+	}
+
+	return &URL{
+		Name: route.Name,
+		URL:  route.Spec.Host,
+	}, nil
+}
+
+// List lists the URLs in an application. The results can further be narrowed
```

Comment:
👍 nice comment 👍 
We should make habit of adding comments like this for every function we write.